Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoEncoderH265.mm

```diff
@@ -541,8 +553,28 @@ - (void)frameWasEncoded:(OSStatus)status
   }
 
   std::unique_ptr<rtc::Buffer> buffer(new rtc::Buffer());
-  if (!webrtc::H265CMSampleBufferToAnnexBBuffer(sampleBuffer, isKeyframe, buffer.get())) {
-    RTC_LOG(LS_INFO) << "Unable to parse H265 encoded buffer";
+  if (_useAnnexB) {
+    if (!webrtc::H265CMSampleBufferToAnnexBBuffer(sampleBuffer, isKeyframe, buffer.get())) {
+      RTC_LOG(LS_INFO) << "Unable to parse H265 encoded buffer";
+    }
```

Comment:
You return early if `H264CMSampleBufferToAnnexBBuffer` fails, should this do the same?

Reply:
Right, let's do the same thing.